poetry_scripts.py

```diff
@@ -20,6 +30,16 @@ def run_tests():
     check_exit_code("docker compose -f docker-compose-test.yaml down")
 
 
+def run_examples():
+    print("Start examples")
+    check_exit_code("python ./examples/mini_tutorial.py")
+    print("After minitutorial")
+    check_exit_code("python ./examples/room_api.py")
+    print("After room_api")
+    check_exit_code("python ./examples/server_notifications.py")
+    print("All examples run without errors")
+
+
```

Comment:
Shouldn't we automate it by running everything from examples? It seems like it will be easy to forget to add new scripts here